glide.lock

```diff
@@ -41,7 +41,7 @@ imports:
   - httputil
```

Comment:
Wondering why is glide.lock changed without any changes in glide.yaml? 
If you have a new dependency, you should add it to glide.yaml first and then do a `glide up -v` to update the glide.lock. I think `tomb.v2` is one such example. I don't see it in glide.yaml. Is there a reason why?